Add VPN configuration details for different cloud providers (#3806)
astronomer/docs: astro/connect-gcp.md

    values={[
        {label: 'VPC peering', value: 'vpc'},
        {label: 'Private Service Connect', value: 'Private-Service-Connect'},

>         {label: 'VPN', value: 'VPN'},

Reviewer: add option to list above

</TabItem>

<TabItem value="VPN">

:::info

> This connection option is available only for dedicated Astro Hosted clusters and Astro Hybrid.

Reviewer 2: ```suggestion
This connection option is not available for standard Astro Hosted clusters. It is available for dedicated clusters and Astro Hybrid.
```

> This connection option is not available for standard Astro Hosted clusters. It is available for dedicated clusters and Astro Hybrid.

Reviewer: The text here is different from the other info banners

> :::info
> 
> This connection option is not available for standard Astro Hosted clusters. It is available for dedicated clusters and Astro Hybrid.
> 
> :::

Reviewer 2: ```suggestion
:::info
This connection option is only available for dedicated Astro Hosted clusters and Astro Hybrid.
:::
```

> Use this connectivity type for access to on-premises resources or inside onther cloud providers.

Reviewer 2: ```suggestion
Use this connectivity type to access on-premises resources or resources in other cloud providers.
```

#### Perequisites

Retrieve the following information about your VPN device or application:

    - Public IP address

>     - Subnet CIDR (multipple if needed) on your side of the connection

Reviewer 2: ```suggestion
    - Subnet CIDR range, multiple if needed, for your side of the connection
```

    - Preferences regarding shared key and BGP usage
    - IKE settings for the tunnel

#### Contact Astronomer support for VPN configuration on Astro side

> Submit all collected details to [Astronomer support](https://cloud.astronomer.io/open-support-request). CRE team will proceed with the required steps. In case of additional questions or clarifications, CRE team will contact you within Support request or let you know about connectivity tests.

Reviewer 2: ```suggestion
Submit all collected details to [Astronomer support](https://cloud.astronomer.io/open-support-request). The Astronomer CRE team will proceed with the required steps. The CRE team will contact you using your support ticket to ask follow-up questions, request clarification, or let you know about connectivity tests.
```
